.gitignore

```diff
@@ -11,3 +11,10 @@ docs/_build
 regparser.egg-info/
 
 .eregs_index/
+
+# eregs output
+/fr_cache.sqlite
+/diff/
+/layer/
+/notice/
+/regulation/
```

Comment:
Right, but we don't expect end users to _keep_ these folders in `.`. Using the `pipeline` command, the output must be explicitly defined (as a final parameter) and using `build_from`, we'd expect `OUTPUT_DIR` setting to be modified. If you've output these files into the current directory, you've likely made a mistake, which we don't want to encourage by adding it as a gitignore. Does that make sense?